Yoast/wordpress-seo
# Removed references to meta_description checks

Removed code used to check for meta_description being hardcoded inside a theme.

_Update (clearify reason for removal)_
We have removed the 'title' analysis before and partly because of WordPress.org not accepting any hardcoded meta description tags in themes it has become more likely that this will not be present as much as it had been.

**Testing**
1. Add the line `<meta name="description" content="My description">` to your `header.php` in one of the themes in your installation.
2. Switch themes (twice if you edited the current theme)
3. Goto the 'General' settings and don't see the following message on the page:
![screen shot 2016-05-30 at 13 45 21](https://cloud.githubusercontent.com/assets/2005352/15648775/0f51066e-266d-11e6-98cf-bea7628b613e.png)

And on other pages don't see:
![screen shot 2016-05-30 at 14 02 40](https://cloud.githubusercontent.com/assets/2005352/15649107/3ddf4e08-266f-11e6-9b2a-2d3327bcb516.png)

See #4650
